app/Console/Commands/FlightawareImport.php

```diff
@@ -0,0 +1,232 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\FeatureToggle;
+use App\RealopsFlight;
+use Carbon\Carbon;
+use GuzzleHttp\Client;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Config;
+use Symfony\Component\Console\Helper\ProgressBar;
+
+class FlightawareImport extends Command {
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'Realops:LoadFromFlightaware';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Flightaware import task. Interactive and VERY SLOW.';
+
+    /**
+     * Execute the console command.
+     */
+    public function handle(): void {
+        if (!FeatureToggle::isEnabled("realops")) {
+            $this->error("Realops is not enabled");
+            return;
+        }
+
+        $this->info('[CollectInformation] Step 1/3: Collecting Information');
+
+        /*
+         * Import Task: Steps
+         * (Planning) 1. Collect Information
+         * (Planning) 2. Create Chunks
+         * (Fetching) 3. Load Chunks
+         */
```

Comment:
I would like to see all of these steps split up into their own functions. That'll make it easier to read and remove the need for this comment

Reply:
Also make it more clear where each step begins/ends